apps/src/lab2/views/dialogs/StartOverDialog.tsx

```diff
@@ -16,18 +16,27 @@ const StartOverDialog: React.FunctionComponent<BaseDialogProps> = ({
   const currentAppName = useAppSelector(
     state => state.lab.levelProperties?.appName
   );
-  const isTextWorkspace =
-    currentAppName && TEXT_BASED_LABS.includes(currentAppName);
+
+  const generaDialogMessage =
```

Comment:
I'm a bit anti- complexity to make typescript happy, so I'd vote for something like:

```
(top of file)
// Lab-specific messages for starting over.
const LAB_SPECIFIC_MESSAGES: {[key: string]: string} = {
  aichat: commonI18n.startOverAichatModelCustomizations(),
};
```

```
(in component)
  const currentAppName =
    useAppSelector(state => state.lab.levelProperties?.appName) || '';

  const textBasedLab =
    currentAppName && TEXT_BASED_LABS.includes(currentAppName);
  const dialogMessage =
    LAB_SPECIFIC_MESSAGES[currentAppName] || textBasedLab
      ? commonI18n.startOverWorkspaceText()
      : commonI18n.startOverWorkspace();
```

Reply:
I also did not know that, thanks for sharing!